lib/operations/db_helpers.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const ReadPreference = require('mongodb-core').ReadPreference;
+
+/**
+ * Ensures provided read preference is properly converted into an object
+ * @param {(ReadPreference|string|object)} readPreference the user provided read preference
+ * @return {ReadPreference}
+ */
+function convertReadPreference(readPreference) {
```

Comment:
to me it looks like this isn't an operation but in fact a utility that can be moved to a more central `utils.js` or something similar? It could even be added to the top level `utils.js` imo